benchmark-overhead/src/test/java/io/opentelemetry/config/Configs.java

```diff
@@ -13,23 +13,9 @@
   RELEASE(
```

Comment:
Is there any need to have more than 1 `TestConfig` ever? Maybe we could get rid of this class and just construct the config object in test.